client/src/components/RatingsAndReviews/Thumbnail.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Thumbnail = ({ photo }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  let modal;
+  if (showModal) {
+    modal = (
+      <div id="review-modal" className="modal-background">
+        <span className="close-modal" onClick={() => setShowModal(false)} role="presentation"><i className="fas fa-times" /></span>
+        <img className="modal-content" src={photo.url} alt="fullsize" />
+        <div id="modal-caption">User photo</div>
+      </div>
+    );
+  }
+
+  return (
+    <span>
+      <img className="thumbnail" src={photo.url} alt="review-attachment" width="100px" role="presentation" onClick={() => setShowModal(true)} />
+      {modal}
```

Comment:
if showModal is false, will {modal} be undefined here?

Reply:
Yes, when modal is undefined, react ignores it and doesn't render anything on the page

Reply:
oh scratch that,  React doesn't care. If undefined it will just not render anything instead of erroring out